### ISSUE TITLE
Please Standardize Definition Format

### ISSUE THREAD

**Issue by neilotte** [oborel/obo-relations#345]
Please format all definitions (IAO_0000115) to capitalize the first word of every definition and to terminate with a period. 

Presently, this is inconsistent in the files.

**Comment by matentzn** [comment]
There are a lot of definitions of this kind:

```
AnnotationAssertion(obo:IAO_0000115 obo:RO_0002177 "a is attached to part of b if a is attached to b, or a is attached to some p, where p is part of b.")
AnnotationAssertion(obo:IAO_0000115 obo:RO_0002176 "c connects a if and only if there exist some b such that a and b are similar parts of the same system, and c connects b, specifically, c connects a with b. When one structure connects two others it unites some aspect of the function or role they play within the system.")
```

Where the definition starts with a variable. It would look odd I would say to capitalise these..

**Comment by neilotte** [comment]
@matentzn Well, then rephrase them. 

For example:
Old: "a is attached to part of b if a is attached to b, or a is attached to some p, where p is part of b.")
New (additions in boldface): "**For two instances, a and b,** a is attached to part of b if a is attached to b, or a is attached to some p, where p is part of b.")

**Comment by matentzn** [comment]
Yeah makes sense to me. I will wait for some pushback (as I am not really and RO editor), but if I don't hear anything by the time I look at this ticket in March, i will implement your suggestion! Thanks.

**Comment by nlharris** [comment]
Can this be closed?

**Comment by wdduncan** [comment]
I like this proposal, but it hasn't had traction with the wider RO community. Maybe we can discuss on a call?
